src/components/lib/libc/musl-1.1.11/arch/i386/bits/alltypes.h

```diff
@@ -0,0 +1,427 @@
+#define _Addr int
```

Comment:
Is this file auto-generated, and not in the main repo? Not sure why it is in the PR.